data/blog/8-week-cadence.md

```diff
@@ -0,0 +1,86 @@
+
+1. Chromium plans to [release a new milestone every **4 weeks**, starting with Chrome 94 on September 21st, 2021.](https://developer.chrome.com/blog/faster-release-cycle/) This release cadence also adds a new Extended Stable option every 8 weeks, which will contain all updated security fixes.
+
+2. The Microsoft Store will [require Chromium-based apps to be no older than within 2 major versions](https://docs.microsoft.com/en-us/windows/uwp/publish/store-policies#102-security) (for example, if the latest released major version of Chromium is 85, any browser based on Chromium must be on at least Chromium version 83 or higher). This rule includes Electron apps.
+
+**Beginning in September 2021, Electron will release a new major stable version every 8 weeks**, to match Chromium's 8 week Extended Stable releases.
+
+Our first release with Chromium Extended Stable will be **Electron 15** on **September 21st, 2021**.
+
+Knowing the a release cadence change will impact other downstream applications, we wanted to let our developer community know as soon as possible. Read on for more details about our 2021 release schedule.
+
+## Moving to Extended Stable
+
+Given that our original Electron 15 release targeted a non-Extended Stable version (Chromium's Extended Stable versions are based on their even-numbered versions), we needed to change our original target release date. However, an Electron app must use the most recent 2 major versions of Chromium to be accepted to the Microsoft Store. With these two requirements, our Releases WG faced a timing dilemma. 
+
+Below is the planned release schedule for 2021:
+
+| Electron | Chrome | Release | Beta to Stable (Weeks) |
+| -------- | ------ | ------- | ----- |
+| E13 | M91 | 2021-May-25 |  12  |
+| E14 | M93 | 2021-Aug-31 |  14  |
+| E15 | M94 | 2021-Sep-21 |  3  |
+| E16 | M96 | 2021-Nov-16 |  8  |
+| E17 | M98 | 2022-Feb-01 |  11  |
+
+Moving Electron 15 to include Chromium M94 would allow app developers to get on the very first Extended Stable version of Chromium - however, it would also shorten the beta-to-stable cycle to only 3 weeks. Yikes!
+
+We've received feedback from the community in the past that our release cycle was already aggressive, and we didn't want to put developers in a position where they felt they weren't given enough time to update their Electron versions.
+
+## Electron 15: Temporary Alpha
+
+To help with this switchover, Electron will offer a temporary **alpha build**, only for the Electron 15 release. This alpha build will allow developers more time to test and plan for an Electron 15 release, with a more stable build than our current nightlies.
+
+The alpha channel build will ship for **Electron 15** on **July 20th, 2021**. It will transition to a beta release on **September 1st, 2021**  with a stable release target of **September 21st, 2021**. With each subsequent Electron release, alphas will be released alongside beta releases for their respective versions.
```

Comment:
```suggestion
The alpha channel build will ship for **Electron 15** on **July 20th, 2021**. It will transition to a beta release on **September 1st, 2021**  with a stable release target of **September 21st, 2021**.
```